magpie-persist/src/main/resources/db/migration/V2__create_static_type_tables.sql

```diff
@@ -73,6 +73,7 @@ create table awsusercredentialreport () INHERITS (aws);
 create table awsssminstance () INHERITS (aws);
 create table orvnshadowaccount () INHERITS (aws);
 create table awswatchloggroup () INHERITS (aws);
+create table awsorganizationscp () INHERITS (aws);
```

Comment:
Please move this to another migration (`V3__blah_blah.sql`) so it doesn't break any existing Magpie installations in use (by OSS users or devs within OR).

Reply:
Updated